### ISSUE TITLE
Communicate melee better in Tutorial 2

### ISSUE THREAD

**Issue by kb173** [lizrad/recursio#183]
Both testers didn't notice the help text suggesting to use melee for killing the enemy. After #177, we could just switch the "KILL" text of the enemy to "MELEE (RB)" after failing with the wall placement once.
